src/ContinuumAbsorption/absorption_metals_bf.jl

```diff
@@ -0,0 +1,43 @@
+metal_bf_cross_sections = let 
+    cross_sections = Dict{Species, Any}()
+    h5open(joinpath(_data_dir, "metal_bf_cross-sections", "bf_cross-sections.h5")) do f
+        T_grid = read(f["logT_min"]) : read(f["logT_step"]) : read(f["logT_max"])
+        ν_grid = read(f["nu_min"]) : read(f["nu_step"]) : read(f["nu_max"])
+        for dataset in f["cross-sections"]
+            #parse the dataset name to a species
+            spec = Species(split(HDF5.name(dataset), "/")[end])
+            σs = read(dataset)
+            cross_sections[spec] = LinearInterpolation((ν_grid, T_grid), σs, extrapolation_bc=Flat())
+        end
+    end
+    cross_sections
+end
+
+"""
+    metal_bf_absorption!(α, νs, T, number_densities)
+
+Adds to α the contributions of bf metal opacities.  Uses precomputed tables from 
+[TOPBase](http://cdsweb.u-strasbg.fr/topbase/topbase.html) for Li, Be, B, C, N, O, F, Ne, Na, Mg, 
+Al, Si, S, Ar, and Ca. Uses tables from 
+[NORAD](https://www.astronomy.ohio-state.edu/nahar.1/nahar_radiativeatomicdata/index.html) for 
+Fe.  For these elements, tables have been procomputed for the neutral and singly ionized species 
+assuming and LTE distribution of energy levels.  See `Korg/data/metal_bf_cross-sections/` for the
+scripts which generate the tables.
+
+Cross sections were computed for 100 K < T < 100,000 K and frequencies corresponding to 
+500 Å < λ < 30,000 Å.  Outside of either of those ranges, flat extrapolation is used (i.e. the 
+extreme value is returned).
+"""
```

Comment:
If we're not including corrections for stimulated emission, I think we need to be explicit about this (since just about every other source of continuum absorption includes a correction for stimulated emission)